src/main/webapp/WEB-INF/tags/instructor/course/courseEditCourseInfo.tag

```diff
@@ -42,8 +50,8 @@
             
             <div class="form-group">
                 <div class=" col-sm-12 align-center">
-                    <input type="submit" class="button" id="btnSaveCourse" name="btnSaveCourse"
-                            style="display:none;" value="Save Changes" onclick="return verifyCourseData();">
```

Comment:
Hmm I removed it because the save button was already there long ago even before there was this function to edit and there is no such verifyCourseData method found in any of the imported js on this page. And the server validates empty string too so I thought I should just remove it.

Should I add a verifyCourseData in the js imported by this page or just leave it as it is now? @kanghj